### PR TITLE
None of the features require a step to enable AFAICT

### PR DESCRIPTION
**Pull request by lamagnifica**
@nedbat

### CODE REVIEW

**Comment by edx-webhook** [comment]
# PR Cover Letter
- [ ] A detailed description of the changes in the body of the PR
- [ ] If available, a link to the relevant JIRA ticket in the body of the PR
  - `[PROJ-XXXX](https://openedx.atlassian.net/browse/PROJ-XXXX)`

# Sandbox
- [ ] Build a sandbox for your branch and add a link here

# Testing
- [ ] i18n
- [ ] RTL
- [ ] Unit, integration, acceptance tests as appropriate
- [ ] Analytics
- [ ] Performance
- [ ] Database migrations are backwards-compatible

# Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [ ] Code review #1
- [ ] Code review #2
- [ ] UI strings/error msgs review
- [ ] UX review
- [ ] Accessibility review
- [ ] Product review

# DevOps assistance
If it's needed, or even if you suspect it might be needed, do any/all these things as early as possible. Don't surprise devops right before merging! For example, touching envs/aws.py or or envs/common.py are dead giveaways that you'll need assistance when pushing your changes out. Things that require devops attention include, but are not limited to:
- configuration changes
- feature flags
- migrations
Ways to get devops attention, pick-and-choose whichever is most appropriate for your specific PR:
- [ ] Tag devops on your PR (`@edx/devops`), either for review (more specific to devops = better) or just a general FYI
- [ ] File a support ticket

FYI: Tag anyone who might be interested in this PR here.

# Post-review
- [ ] Squash commits

File: en_us/install_operations/source/dogwood.rst [hunk 1]
```diff
@@ -206,6 +194,8 @@ The ``migrage.sh`` script implements this process.  You may need to understand
 this process if your installation is customized in some way, or if you need to
 diagnose problems during the upgrade.
 
+.. Ned is the name of the script migrate or migrage?^^
```
Comment (nedbat) [inline en_us/install_operations/source/dogwood.rst]: "migrate" oops!

File: en_us/links/links.rst [hunk 2]
```diff
@@ -107,6 +107,8 @@
 
 .. _Developing on Devstack: https://github.com/edx/edx-platform/wiki/Developing-on-the-edX-Developer-Stack
 
+.. _forum migration described on the Open edX wiki: https://openedx.atlassian.net/wiki/display/TNL/Migrating+the+Discussion+Forums+to+Support+Teams+Discussion+Filtering
```
Comment (nedbat) [inline en_us/links/links.rst:110]: I don't know what the policy is for links, but some of these are very local, and would be easier to deal with in the doc file itself.  For example, the next link below is "init file", that surely has no meaning outside of its particular usage someplace else.

File: en_us/install_operations/source/dogwood.rst [hunk 3]
```diff
@@ -20,27 +20,15 @@ What's Included in Dogwood
...
+teams, and developers. See the release notes for the
+:ref:`openreleasenotes:Open edX Dogwood Release` for more details.
 
 ******************************
 What is the Dogwood Git Tag?
 ******************************
 
-The Git tag for the Dogwood release is ``named-release/dogwood.rc3``. You use this
-tag to identify the version of Open edX code that is the Dogwood release.
+The Git tag for the Dogwood release is ``named-release/dogwood.rc3``. You use
```
Comment (nedbat) [inline en_us/install_operations/source/dogwood.rst]: We're going to have to update these to the final tag (without "rc3"), but we can do that in another pass.

**Comment by nedbat** [comment]
:+1:

File: en_us/links/links.rst (hunk 2 above)
Comment (lamagnifica) [inline en_us/links/links.rst:110]: I don't disagree. We changed to the common links file just before Mark left, so we are still trying to get a strategy in place.

**Pull request merged**